### PR TITLE
Update to WildFly 23.0.0.Final

### PR DESCRIPTION
**Pull request by WolfgangHG**

### CODE REVIEW

File: wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/README.txt [hunk 1]
```diff
@@ -19,8 +19,8 @@ If you don't use entity beans, you can delete "persistence.xml".
 ==========================
 
 JSF:
-The web application is prepared for JSF by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
-In case you don't want to use JSF, simply delete this file.
+The web application is prepared for JSF 2.3 by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
+In case you don't want to use JSF, simply delete this file and "src\main\webapp\beans.xml" and "src\main\java\Jsf23Activator.java"
```
Comment (boris-unckel) [inline wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/README.txt:23]: it's "src\main\webapp\WEB-INF\beans.xml" 
I personally prefer the / for directories. Is there any convention here?

File: wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/__rootArtifactId__-web/src/main/webapp/WEB-INF/beans.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="
+      http://xmlns.jcp.org/xml/ns/javaee
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
```
Comment (boris-unckel) [inline wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/__rootArtifactId__-web/src/main/webapp/WEB-INF/beans.xml:5]: As WF23 adresses Jakarata EE 8, this should be [https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives](https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives)
`xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd" bean-discovery-mode="annotated" version="2.0"`

File: wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/beans.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="
+      http://xmlns.jcp.org/xml/ns/javaee
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
```
Comment (boris-unckel) [inline wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/beans.xml:5]: As WF23 adresses Jakarata EE 8, this should be [https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives](https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#declaring_selected_alternatives)
`xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd" bean-discovery-mode="annotated" version="2.0"`

File: wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/README.txt [hunk 4]
```diff
@@ -19,8 +19,8 @@ If you don't use entity beans, you can delete "persistence.xml".
 ==========================
 
 JSF:
-The web application is prepared for JSF by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
-In case you don't want to use JSF, simply delete this file.
+The web application is prepared for JSF 2.3 by bundling an empty "faces-config.xml" in "src\main\webapp\WEB-INF".
+In case you don't want to use JSF, simply delete this file and "src\main\webapp\beans.xml" and "src\main\java\Jsf23Activator.java"
```
Comment (boris-unckel) [inline wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/README.txt:23]: it's "src\main\webapp\WEB-INF\beans.xml"
I personally prefer the / for directories. Is there any convention here?

File: wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/README.txt (hunk 1 above)
Comment (WolfgangHG) [reply wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/README.txt]: Changed.

File: wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/__rootArtifactId__-web/src/main/webapp/WEB-INF/beans.xml (hunk 2 above)
Comment (WolfgangHG) [reply wildfly-jakartaee-ear-archetype/src/main/resources/archetype-resources/__rootArtifactId__-web/src/main/webapp/WEB-INF/beans.xml]: Changed. Thanks for spotting this!

File: wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/beans.xml (hunk 3 above)
Comment (WolfgangHG) [reply wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/beans.xml]: Changed.

File: wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/README.txt (hunk 4 above)
Comment (WolfgangHG) [reply wildfly-jakartaee-webapp-archetype/src/main/resources/archetype-resources/README.txt]: Changed.

**Pull request merged**